### PR TITLE
implemented DateTimeInput instead of DateInput if fieldOptions.showTime

### PR DESCRIPTION
**Pull request by farandal**
admin_gdtlk_co___streams_5b4e5e36eb0cd700258792c0

### CODE REVIEW

File: src/AutoAdmin.tsx [hunk 1]
```diff
@@ -236,7 +237,11 @@ const attributeToInput = (input: AutoAdminAttribute) => {
     case Boolean:
       return <BooleanInput label={input.label} source={input.attribute} options={input.fieldOptions} />;
     case Date:
-      return <DateInput label={input.label} source={input.attribute} options={input.fieldOptions} />;
+      return input.fieldOptions && input.fieldOptions.showtime ? (
```
Comment (golastra) [inline src/AutoAdmin.tsx]: Let's try to use `camelCase` for `showtime`

**Comment by farandal** [comment]
https://github.com/marmelab/react-admin/blob/master/docs/Inputs.md#datetimeinput

**Comment by baytelman** [comment]
Screen grab! How does it look in our admin? Is it clear about the timezone?

**Comment by farandal** [comment]
@golastra @baytelman  maybe we can try this one, I have been using it and it's amazing, it also uses material ui.

**Pull request merged**